osbs/build/manipulate.py

```diff
@@ -50,6 +50,17 @@ def remove_plugin(self, plugin_type, plugin_name):
                 self.dock_json[plugin_type].remove(p)
                 break
 
+    def add_plugin(self, plugin_type, plugin_name, args_dict):
+        """
+        if config has plugin, override it, else add it
```

Comment:
In the case of modifying a plugin, I think it'd make sense to replace it in place, instead of appending to end of list.